lua/houdini.lua

```diff
@@ -114,6 +116,10 @@ function M.setup(opts)
     if opts then
         config = assert(vim.tbl_deep_extend('force', defaults, opts))
 
+        for _, excluded_filetype in ipairs(config.excluded_filetypes)  do
+          excluded_filetypes_set[excluded_filetype] = true
```

Comment:
Its a nitpick, but would you be so kind to adopt the formatting to match the 4 spaces indentation of the rest of the file :slightly_smiling_face:

Reply:
done